### PR TITLE
Docs updated for CSI-PowerFlex driver v2.7.0

### PR DESCRIPTION
**Pull request by ashleyvjoy**
# Description
Updated release to 2.7.0
Updated feature list and release notes

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| https://github.com/dell/csm/issues/743 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/offline/_index.md [hunk 1]
```diff
@@ -222,9 +222,9 @@ Preparing a offline bundle for installation
    dellemc/csi-unity:v2.3.0 -> localregistry:5000/csi-operator/csi-unity:v2.3.0
    dellemc/csi-unity:v2.4.0 -> localregistry:5000/csi-operator/csi-unity:v2.4.0
    dellemc/csi-unity:v2.5.0 -> localregistry:5000/csi-operator/csi-unity:v2.5.0
-   dellemc/csi-vxflexos:v2.4.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.4.0
    dellemc/csi-vxflexos:v2.5.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.5.0
    dellemc/csi-vxflexos:v2.6.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.6.0
+   dellemc/csi-vxflexos:v2.4.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.7.0
```
Comment (VamsiSiddu-7) [inline content/docs/csidriver/installation/offline/_index.md:227]: this should be v2.7.0

File: content/docs/csidriver/installation/offline/_index.md [hunk 2]
```diff
@@ -249,9 +249,9 @@ Preparing a offline bundle for installation
    changing: dellemc/csi-unity:v2.3.0 -> localregistry:5000/csi-operator/csi-unity:v2.3.0
    changing: dellemc/csi-unity:v2.4.0 -> localregistry:5000/csi-operator/csi-unity:v2.4.0
    changing: dellemc/csi-unity:v2.5.0 -> localregistry:5000/csi-operator/csi-unity:v2.5.0
-   changing: dellemc/csi-vxflexos:v2.4.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.4.0
    changing: dellemc/csi-vxflexos:v2.5.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.5.0
    changing: dellemc/csi-vxflexos:v2.6.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.6.0
+   changing: dellemc/csi-vxflexos:v2.4.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.7.0
```
Comment (VamsiSiddu-7) [inline content/docs/csidriver/installation/offline/_index.md:254]: same here

**Review by rsedlock1958 — APPROVED** [review]
No edits required

**Review by rsedlock1958 — APPROVED** [review]
No edits required

File: content/docs/csidriver/installation/offline/_index.md (hunk 1 above)
Comment (ashleyvjoy) [reply content/docs/csidriver/installation/offline/_index.md]: Addressed the comment

File: content/docs/csidriver/installation/offline/_index.md (hunk 2 above)
Comment (ashleyvjoy) [reply content/docs/csidriver/installation/offline/_index.md]: Addressed the comment

**Review by VamsiSiddu-7 — APPROVED** [review]

**Review by KshitijaKakde — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**